src/views/Organizations/OrganizationResourceTransfer.vue

```diff
@@ -28,26 +28,25 @@ resourceAvailability.value = await store.dispatch(
 
 const postQuantityTransfer = async (data) => {
   try {
-    await store.dispatch("resources/resourceAvailabilityTransfer", data);
+    await store.dispatch("organizations/transferResourceFromOrg", data);
     snackbarProvider.showSuccessSnackbar("Successfully transfered!");
-    router.push(`/users/${ownerId}`);
+    router.push(`/organization/${organizationId}`);
```

Comment:
missed an 's'? Maybe we should use constants?
```suggestion
    router.push(`/organizations/${organizationId}`);
```

Reply:
its without "s" as its a single organization, also by id

Reply:
I still think it should be organization**s**/{organzationId} because this way we keep our paths consistent. If we have an id we know it's a single organization. This is the same pattern we use for the backend endpoints.